src/api/dokumentApi.ts

```diff
@@ -0,0 +1,10 @@
+import { Dokument, DokumentIdType } from '~pages/saksbehandling/dokument/Dokument';
+
+import apiClient, { ApiClientResult } from './apiClient';
+
+export async function hentDokumenter(id: string, idType: DokumentIdType): Promise<ApiClientResult<Dokument[]>> {
+    return apiClient({
+        url: `/dokumenter?id=${id}&idType=${idType}`,
```

Comment:
Jeg er vant med at når man bruker query parametre, så er det fordi de er optional. Men det er de ikke her? Så hvorfor endre stil fra path til query parametre?

Reply:
Enig i at det føles litt rart siden det ikke er snakk om "filtrering", men jeg og @jacob-meidell diskuterte det litt nå, og kom egentlig til at den beste REST-fulle måten å gjøre det på hadde vært å legge dokumentene under de forskjellige ressursene; `/sak/{sakId}/dokumenter` osv osv. Det blir dessverre ganske upraktisk, siden vi da nok hadde endt opp med å duplisere funksjonaliteten for hver av de tingene som skal ha dokumenter.

Tenkte også på f.eks. `dokumenter/{idType}/{id}`, men det er jo heller ikke så veldig REST-ful. Har du noen andre forslag, @hestad ?

Reply:
Greit for meg å duplisere Route-funksjonene, men beholde en enkelt service :shrug: